eng/pipelines/dotnet-buildtools-prereqs-centos.yml

```
@@ -25,3 +25,4 @@ stages:
     internalProjectName: ${{ variables.internalProjectName }}
     publicProjectName: ${{ variables.publicProjectName }}
     linuxAmdBuildJobTimeout: 210
+    linuxAmdTestJobTimeout: 210
```

```suggestion
    linuxArmBuildJobTimeout: 210
```

The same parameter needs to be added to: https://github.com/dotnet/dotnet-buildtools-prereqs-docker/blob/31c08723eca277df5b0e044b261d438a7578f5c7/eng/pipelines/dotnet-buildtools-prereqs-centos-pr.yml#L24-L27

---

OK, I was wondering where I needed to add that. I will make these changes.